### DISCUSSION TITLE
Edit component on doubleclick

### DISCUSSION THREAD

**Discussion by xszaboj** [formio/formio.js#5282] [Q&A]
Hi, is there any way to edit component on double click rarther than click edit button?
![image](https://github.com/formio/formio.js/assets/3457401/b5e7969d-1665-4915-ac72-2cb1a62eed59)

Thanks for any ideas

**Comment by dinbtechit** [comment]
I had the same question.. also, I would like to see the editModal show up on the right of the page as a drawer.

**Reply by xszaboj** [reply]
Any ideas how to approach this? I've tried few things but haven't found any documentation on this or what events/listeners to hack...

**Reply by dinbtechit** [reply]
I haven't tried it. But every single element with `ref['component']` has a hidden `div.component-btn-group` which contains the buttons. Basically we can write a javascript to simulate the button click.


**Untested code.**

```js

document.querySelector('div.component-btn-group > div.btn.btn-xxs.btn-default.component-settings-button.component-settings-button-edit-json').click();
```

**Reply by xszaboj** [reply]
Ok thanks for help. I'll try it

**Reply by dinbtechit** [reply]
let me know if you were successful! Cheers!

**Reply by DSBRBrasil** [reply]
HI! 
do you achived ?
